src/utils/data.js

```diff
@@ -1,7 +1,7 @@
 export const filterOptions = [
     { id: "0", text: "All datasets" },
     { id: "1", text: "My allocations" },
-    { id: "2", text: "2X Stake" },
+    { id: "2", text: "Publisher 2X stake" },
```

Comment:
In terms of length it sounds good, but it's quite ambiguous.
For example if you are on the DCV and have published 2x selected what does that mean, is it 2x DCV?

Reply:
Publisher 2x specifically means that you get your veOCEAN effectively doubled during the reward calculations every week.

It's [well documented in the docs](https://docs.oceanprotocol.com/~/revisions/E1xWAzXFffFRpE9EP7w4/rewards/df-volumedf#what-are-publisher-rewards) and in blog posts


When you select this, users are able to see that these assets normally have high veOCEAN allocated to them.
![image](https://github.com/oceanprotocol/df-web/assets/69865342/c1c2fbb7-5466-4cb5-b1b2-6726a6f81b51)

[To improve visualizing this bonus]
1. If the user clicked on the row or hovered, it could be shown that:
- Total Allocation (veOCEAN)
- Publisher Allocation (veOCEAN)
- Publisher Bonus (veOCEAN *2)

2. Publisher Bonuses could also be visualized via a DF round summary chart.

TLDR; Improving this visualization at the moment is not important.